pkg/xds/envoy/routes/reset_tags_configurer.go

```diff
@@ -0,0 +1,17 @@
+package routes
+
+import envoy_api_v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
+
+func ResetTags() RouteConfigurationBuilderOpt {
```

Comment:
How about `ResetTagsHeader()`?

Reply:
Also add doc here that this tag is meant to use internally, so we don't want to expose it to destination application.